packages/template-ui/src/components/ContentImage.vue

```diff
@@ -1,18 +1,23 @@
 <template>
   <b-link @click="goToContent()">
-    <b-card class="rounded-lg">
+    <b-card
+      class="content-image rounded-lg"
+      :class="{ 'mx-auto': bundleKind }"
```

Comment:
This is taking advantage of the `bundleKind` being only `"simple"` or `null` today, but is not future-proof. Please use the `isSimpleBundle` getter instead of using the `bundleKind` state directly.